docs/src/understand/single-sign-on/azure/main.md

```diff
@@ -1,58 +1,76 @@
-# How to set up SSO integration with Microsoft Azure
+# How to set up SSO integration with Microsoft Entra ID
+
+## Purpous and Scope
+This document is for current and perspective users of the wire cloud, who want to manage their users with Microsoft Entra ID, in Azure.
 
 ## Preprequisites
 
-- <http://azure.microsoft.com> account, admin access to that account
-- See also {ref}`sso-generic-setup`.
+Before you begin, we are going to assume you have the following:
+- An account on <http://azure.microsoft.com>
+  * admin access to that account, sufficient to add an application, and manage users.
+- A team on https://app.wire.com/ or another wire backend, and admin access to that team.
+
+## Process
 
-## Steps
+### Creating a new application
+Go to <https://portal.azure.com/>, and login. You should be brought to the 'Microsoft Azure' home page.
```

Comment:
```suggestion
Go to [portal.azure.com](https://portal.azure.com/), and login. You should be brought to the 'Microsoft Azure' home page.
```

Reply:
AFAIK `<...>` renders no link in Markdown.

Reply:
https://docs.wire.com/understand/single-sign-on/azure/main.html disagrees with you, but sure. :)